ConnectorRhino/ConnectorRhino8/ConnectorRhino8.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0-windows</TargetFramework>
+    <Title>SpeckleRhino</Title>
+    <Description>Description of SpeckleRhino</Description>
+    <TargetExt>.rhp</TargetExt>
+    <RhinoMacLauncher>/Applications/Rhino 8.app</RhinoMacLauncher>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AssemblyName>SpeckleConnectorRhino</AssemblyName>
+    <Configurations>Debug;Release;Debug Mac;Release Mac</Configurations>
+    <RootNamespace>SpeckleRhino</RootNamespace>
+    <Product>ConnectorRhino8</Product>
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
+
+    <!--NOTE-->
+    <!--
+    Since Rhino uses mono we build for win-x64 also on mac
+    BUT we then have to manually copy the dylib for Avalonia and Microsoft.Data.Sqlite
+    They are loaded from the nuget mac folder
+    REMEMBER to update its numbers if updating the nugets
+    -->
+    <PlatformTarget>x64</PlatformTarget>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
Is this part of this ticket? Do we need a subtask?

Reply:
Already created a subtask to look into the Mac side of things (which is currently not working, possibly related to this).

I'll resolve this conversation for now.

Reply:
Hi @AlanRynne. Is there a resolution on this? I am trying to use the `Speckle.Objects.Converter.Rhino8` package on macOS, but the nuget states it will not install as it requires "net7.0-windows7.0"

(BTW, Speckle.Objects.Converter.Rhino7 installs just fine...)